components/src/Modal/Modal.story.js

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { Modal as NDSModal, Button, Form, Input } from "../index";
+
+const env = process.env.NODE_ENV;
+
+if (env !== "test") NDSModal.setAppElement("#root");
+
+const envProps = {
+  ariaHideApp: env === "test" ? false : undefined
+};
+
+const Modal = props => <NDSModal {...envProps} {...props} />;
+
+const primaryButton = { label: "Primary Action", onClick: () => {} };
+
+const secondaryButtons = [{ label: "Secondary Action", onClick: () => {} }];
+
+// Modal.setAppElement("#root")
+
+class ModalExample extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      isOpen: false
+    };
+
+    this.openModal = this.openModal.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+  }
+
+  openModal() {
+    this.setState({ isOpen: true });
+  }
+
+  closeModal() {
+    this.setState({ isOpen: false });
+  }
+
+  render() {
+    const { isOpen } = this.state;
+    return (
+      <div>
+        <Button onClick={this.openModal}>Open Modal</Button>
+        <Modal
+          title="Edit Profile"
+          onRequestClose={this.closeModal}
+          primaryButton={{ label: "Submit", type: "submit", form: "myForm" }}
+          secondaryButtons={[{ label: "Cancel", onClick: this.closeModal }]}
+          isOpen={isOpen}
+          maxWidth="456px"
+        >
+          <Form id="myForm" mb="x2">
+            <Input name="name" id="name" labelText="Name" />
+            <Input type="number" name="age" id="age" labelText="Age" />
+          </Form>
+        </Modal>
+      </div>
+    );
+  }
+}
+
+storiesOf("Modal", module)
```

Comment:
Unused param `module`

Reply:
module is required for storybook to have hot-refreshing: https://github.com/storybookjs/storybook/issues/2022